scripts/utils/integration-tests

```diff
@@ -199,6 +205,18 @@ function report {
 
 # finds the files and runs the tests
 function run_tests {
+  run_tests_one_by_one
+}
+
+function run_all_tests {
```

Comment:
this isnt used, was it a test to see if all tests at once are faster?is it the case? if not used, should it be removed?

Reply:
I'd like to keep it. I know it will be faster  :)